stable/restore/values.yaml

```diff
@@ -81,12 +81,16 @@ restore:
   # name of the existing database being restored 
   target: "demo"
 
-  # source
-  # Can be one of:
+  # The source can be one of:
   # * the name of a backupset on a hotcopy-SM's backup disk;
   # * the metaname ':latest'
   # * the URL of a downloadable TAR file of a hotcopy backupset
   # * stream:URL of a downloadable TAR file of a exact copy of a NuoDB archive
```

Comment:
This comment about "stream" is here but not in the readme.  Also do we have to mention anything abou the strip levels we used to supply when downloading a tar?

Reply:
I will update the README that the URL should point either to backupset or an exact copy of NuoDB archive (_stream_). 
Currently, we don't expose strip levels in the restore chart (default of 1 is used). I will open another issue about this which will fix in separate PR.